### PR TITLE
chore: add SDK version in resources

### PR DESCRIPTION
**Pull request by vincentdelmundo**

### CODE REVIEW

File: inappmessaging/build.gradle [hunk 1]
```diff
@@ -10,6 +10,7 @@ apply plugin: 'kotlin-kapt'
 
 android {
   defaultConfig {
+    resValue 'string', 'inappmessaging__version', project.version
```
Comment (donnie-jp) [inline inappmessaging/build.gradle:13]: What's this for? The analytics sdk_info object?

Comment (vincentdelmundo) [reply inappmessaging/build.gradle:13]: yes, for IAM to be included in supported modules

Comment (donnie-jp) [reply inappmessaging/build.gradle:13]: On iOS we're able to do this solely from the Analytics SDK without requiring the setting of any variables in the SDKs-to-be-tracked.  Is that possible on android?

**Review by donnie-jp — APPROVED** [review]
👍

File: inappmessaging/build.gradle (hunk 1 above)
Comment (vincentdelmundo) [reply inappmessaging/build.gradle:13]: The only other way is to use the package manager, but the Analytics SDK should know all the package name (hard-coded) of the supported modules.

Comment (vincentdelmundo) [reply inappmessaging/build.gradle:13]: Based on additional investigation. Using package manager won't work.
This approach is the only way for now.

**Pull request merged**